### ISSUE TITLE
White flash on startup

### ISSUE THREAD

**Issue by tadaaagames** [subsoap/defos#108]
I've found that on startup of the application I get a white flash for 4-5 frames when using Defos that doesn't happen on vanilla Defold.

Is this something that can be fixed/worked around or is part of the loading process?

**Comment by subsoap** [comment]
It would be worth making an issue https://github.com/defold/editor2-issues/issues here about this. I assume on their vanilla engine they have the clear color set to black, but in the extended version it's set to white.

**Comment by tadaaagames** [comment]
Yep, will do.

Thanks!

**Comment by britzl** [comment]
> I assume on their vanilla engine they have the clear color set to black, but in the extended version it's set to white.

I don't think this is true anymore (or if it ever was). We did however make a change maybe half a year ago where the engine sets the clear color earlier during initialization than it did before.

Note that I can't recall seeing a white flash on startup in projects using DefOS though). @gianmichelemariani are you still experiencing this problem? Which OS?

**Comment by gianmichelemariani** [comment]
Let me check again. I haven't used defos in a while :)

**Comment by gianmichelemariani** [comment]
Apology for the delay. 
I've tested this, without defos and with all the latest launchscreen storyboard, etc. Version is 1.2.181 beta (but I think it happens in other version too) and it's a debug build.

What is happening here is that on launch of the application, the storyboard part is shows (I have just a white background), then there is a flash to black, and then the render kicks in (I have a clear background color to white and a logo animated).

I'm attaching a video for reference. You'll need to go through it frame by frame to see it (quicktime is good).

https://user-images.githubusercontent.com/46694168/112043808-f6d11f00-8b40-11eb-8245-66d6b8453cd7.mov

What might be causing this?

**Comment by subsoap** [comment]
@gianmichelemariani Do you have any other native extensions enabled?

**Comment by gianmichelemariani** [comment]
Just these 2

<img width="653" alt="Screenshot 2021-03-23 at 17 09 05" src="https://user-images.githubusercontent.com/46694168/112187997-825ab680-8bfa-11eb-8949-cca694f66da6.png">
